src/composables/actions.ts

```diff
@@ -51,5 +51,7 @@ export const ACTION_TO_FUNC = {
   decrementLabel: applyLabelOffset(-1),
   incrementLabel: applyLabelOffset(1),
 
+  mergeNewPolygon: () => {}, // no-op
```

Comment:
Should add a brief comment as to why this is a no-op

Reply:
Added // acts as a modifier key rather than immediate effect, so no-op